index.bs

```diff
@@ -7454,15 +7428,22 @@ In effect, where <var ignore>x</var> is a Number value,
 </div>
 
 
-<h4 id="es-float">float</h4>
+<h4 id="es-decimal-types">Decimal types</h4>
```

Comment:
Floats are not decimal types. Any reason not to stick with the existing structure here?

Reply:
It’s to allow `[Unrestricted]` to refer to the `float` and `double` section as a whole.

Reply:
Can't we just say float or double? We don't do this for integers types.

Reply:
Actually, [`[Clamp]`](https://heycam.github.io/webidl/#Clamp) does do that.

Reply:
Fair, so let's add "float types" in a similar fashion, but not establish a new outline for the document.